dagstore_control.go

```diff
@@ -120,10 +131,11 @@ func (d *DAGStore) control() {
 				s.wAcquire = s.wAcquire[:0] // clear acquirers.
 			}
 
+			// TODO What about all those who have already acquired
```

Comment:
This part of the code doesn't know which kind of error occurred. Could be that the index has disappeared, but active acquirers already have the index so we shouldn't interrupt them. If this is a physical error (e.g. shard data was physically deleted, or corrupted), we'll leave to the `ShardAccessor` (and the `ReadBlockstore`) to fail at some point. At that stage, the caller will call `ShardAccessor#Close` and eventually all active references will be released.

Reply:
Documented in #51.

Reply:
Makes sense.